### ISSUE TITLE
Change Build runs macos-lastest by macos-15

### ISSUE THREAD

**Issue by AhmedAmineZr** [Orange-OpenSource/ouds-flutter#344]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Any

### OS Version

Any

### App version

0.6.0

### Describe the issue

Build iOs in GitHub is not working report Failed : 
`Failed to build iOS app
Uncategorized (Xcode): Unable to find a destination matching the provided destination specifier:
		{ generic:1, platform:iOS }`

### Expected Behavior

The build will working and finish with success 
Eventual changes: is to change macos-lastest by macos-15

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
